### DISCUSSION TITLE
does the modelviewer.dev/editor have a github page?

### DISCUSSION THREAD

**Discussion by zntr1** [google/model-viewer#2864] [Q&A]
Hey guys,

I wanted to check out the AR QR-Code logic used to create a QR-Code to scan with the phone. However I couldn't find the source code for it. Any help?

Thanks

**Comment by elalish** [comment]
It's all  in here: https://github.com/google/model-viewer/tree/master/packages/space-opera/src/components/mobile_view

The key is it uses `piping-server`.
